src/forms/reduxForm/reduxTextInput.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
ok. Thanks for trying that out. I think I wanted to see if the snapshot rendered the correct html all the way through, but you're right - we do have another test for `TextInput` that does that. I'll clean this up.